requirements.txt

```diff
@@ -1,3 +1,3 @@
 prefect>=2.8.2
 distributed==2022.2.0; python_version < '3.8'
-distributed>=2022.5.0,<=2023.3.1; python_version >= '3.8' # upper bound due to issue with get_client starting in 2023.3.2 - https://github.com/dask/distributed/issues/7763
+distributed>=2022.5.0,<=2023.8.0; python_version >= '3.8' # upper bound due to issue with get_client starting in 2023.3.2 - https://github.com/dask/distributed/issues/7763
```

Comment:
cc @chrisguidry this upper bound should be removed entirely per https://github.com/PrefectHQ/prefect-dask/issues/115

Reply:
Oh great call, I'll close this one out for now